src/main/java/net/dries007/tfc/objects/items/ItemOverriddenGlassBottle.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.objects.items;
+
+import java.util.Optional;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemGlassBottle;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.World;
+import net.minecraftforge.fluids.FluidActionResult;
+import net.minecraftforge.fluids.FluidUtil;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import net.minecraftforge.fluids.capability.IFluidTankProperties;
+import net.minecraftforge.items.IItemHandler;
+
+import mcp.MethodsReturnNonnullByDefault;
+import net.dries007.tfc.util.Helpers;
+
+import static net.minecraftforge.items.CapabilityItemHandler.ITEM_HANDLER_CAPABILITY;
+
+@MethodsReturnNonnullByDefault
+@ParametersAreNonnullByDefault
+
+public class ItemOverriddenGlassBottle extends ItemGlassBottle
+{
+    public ItemOverriddenGlassBottle()
+    {
+        setTranslationKey("glassBottle");
+        setRegistryName(Items.GLASS_BOTTLE.getRegistryName());
```

Comment:
These should both be part of the registration call, i.e. `new ItemGlassBottleTFC().setRegistryName()...`

Reply:
My 2c is - I think configuring item in ItemTFC harms readability. Having self-sufficient object upon construction is generally good practice. 
On the other hand I see you have established approaches, so I made change as you requested.

Reply:
A couple reasons:
 - registry name and translation key need to be unique, so putting it in the constructor forces that class to only be used for exactly one item (not a problem with this case, but for many items this will apply)
 - All of vanilla item constructors do this, which is what this is copied from. This way it's easier to verify that we've not missed something if it looks the same.